Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -333,7 +333,7 @@ def bump_formula_pr
 
     ohai "brew update-python-resources #{formula.name}"
```

Comment:
```suggestion
```

This line probably isn't needed anymore if we pass `silent: args.quiet?` to `PyPI.update_python_resources!` (whereas previously we always passed `silent: true`)

Example output without PyPI dependencies:

```console
$ brew bump-formula-pr helmfile --version 0.125.3 --dry-run --write --force
...
==> Downloading https://github.com/roboll/helmfile/archive/v0.125.3.tar.gz
==> replace /https:\/\/github\.com\/roboll\/helmfile\/archive\/v0\.125\.2\.tar\.gz/ with "https://github.com/roboll/helm
==> replace "72d3b5646459408b8096c94408ea5fdf72c335575b99e3c0072fe81320b98c6c" with "2ca6b039c72414eaf2ffe5848f61320f8ae
==> brew audit helmfile.rb
...
```

With PyPI dependencies:

```console
$ brew bump-formula-pr jc --version 1.13.4 --dry-run --write --force
...
==> Downloading https://files.pythonhosted.org/packages/e3/b9/7878a4f71c873c7d67f39615086f1c8315740534b25eddc1a4f75f3148
==> replace /https:\/\/files\.pythonhosted\.org\/packages\/2b\/1e\/179eea9186313bcff8dc3405ecd4615043ea615681519a9638d79
==> replace "b5ebb419b3b5d3cd95a166b4f156c7986235e983d5c6bf21aa9d57586e211f78" with "45480ac3d399f70b57d8cc97a6795ea875a
==> Retrieving PyPI dependencies for "jc==1.13.4"...
==> Getting PyPI info for "pygments==2.6.1"
==> Getting PyPI info for "ruamel-yaml==0.16.10"
==> Getting PyPI info for "ruamel-yaml-clib==0.2.0"
==> Getting PyPI info for "xmltodict==0.12.0"
==> Updating resource blocks
==> brew audit jc.rb
...
```